.ci/Jenkinsfile

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env groovy
+
+@Library('apm@current') _
+
+pipeline {
+  agent { label 'ubuntu-18 && immutable' }
+  environment {
+    REPO = "go-ucfg"
+    BASE_DIR = "src/github.com/elastic/${env.REPO}"
+    JOB_GIT_CREDENTIALS = "f6c7695a-671e-4f4f-a331-acdce44ff9ba"
+    PIPELINE_LOG_LEVEL = 'INFO'
+  }
+  options {
+    timeout(time: 1, unit: 'HOURS')
+    buildDiscarder(logRotator(numToKeepStr: '20', artifactNumToKeepStr: '20', daysToKeepStr: '30'))
+    timestamps()
+    ansiColor('xterm')
+    disableResume()
+    durabilityHint('PERFORMANCE_OPTIMIZED')
+    rateLimitBuilds(throttle: [count: 60, durationName: 'hour', userBoost: true])
+    quietPeriod(10)
+  }
+  triggers {
+    issueCommentTrigger('(?i)(.*(?:jenkins\\W+)?run\\W+(?:the\\W+)?tests(?:\\W+please)?.*|^\\/test$)')
+  }
+  stages {
+    stage('Checkout') {
+      steps {
+        deleteDir()
+        gitCheckout(basedir: "${BASE_DIR}")
+        stash allowEmpty: true, name: 'source', useDefaultExcludes: false
+      }
+    }
+    stage('Check') {
+      steps {
+        withGithubNotify(context: 'Check') {
+          cleanup()
+          withMageEnv(version: '1.14'){
+            dir("${BASE_DIR}"){
+              sh(label: 'go-licenser', script: '.ci/goinstall.sh github.com/elastic/go-licenser@master')
+              sh(label: 'go-licence-detector', script: '.ci/goinstall.sh go.elastic.co/go-licence-detector')
```

Comment:
you can install the packages with withMageEnv

```
withMageEnv(version: '1.14', packages: ['github.com/elastic/go-licenser@master', 'go.elastic.co/go-licence-detector']){
}
```

Reply:
That's actually a good point, I just migrated from travis to avoid major improvements, besides of the junit support. So I can give a go now